### PR TITLE
Cleanup via rubocop

### PR DESCRIPTION
**Pull request by atz**
We had rubocop YAML files in the repo, but we didn't actually include rubocop as a development dependency.  No surprise then, the code was not particularly adherent to any defined style.  

This resolves a great number of `Lint`, `Style` and `Layout` issues and removes useless code and indirection.  I declined to employ a couple corrections that might be more problematic, but we could opt to address them subsequently.

Note the couple outstanding citations for `Security/Open` are from using Kernel.open (in spec).  Probably could be converted to `File.open` if we care.

### CODE REVIEW

File: lyber-core.gemspec [hunk 1]
```diff
@@ -1,34 +1,35 @@
...
+  s.version     = '4.1.3'
   s.licenses    = ['Apache-2.0']
   s.platform    = Gem::Platform::RUBY
-  s.authors     = ["Alpana Pande","Bess Sadler","Chris Fitzpatrick","Douglas Kim","Richard Anderson","Willy Mene","Michael Klein", "Darren Weber","Peter Mangiafico"]
-  s.email       = ["sul-devops-team@lists.stanford.edu"]
-  s.homepage    = "http://github.com/sul-dlss/lyber-core"
-  s.summary     = "Core services used by the SUL Digital Library"
-  s.description = "Contains classes to make http connections with a client-cert, use Jhove, and call Suri\n" +
-                    "Also contains core classes to build robots"
+  s.authors     = ['Alpana Pande', 'Bess Sadler', 'Chris Fitzpatrick', 'Douglas Kim', 'Richard Anderson', 'Willy Mene', 'Michael Klein', 'Darren Weber', 'Peter Mangiafico']
```
Comment (ndushay) [inline lyber-core.gemspec]: hahahaha - only @peetucket still works here ...

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -5,9 +5,9 @@
 Coveralls.wear!
 
 SimpleCov.formatter = SimpleCov::Formatter::MultiFormatter.new([
-  SimpleCov::Formatter::HTMLFormatter,
-  Coveralls::SimpleCov::Formatter
-])
+                                                                 SimpleCov::Formatter::HTMLFormatter,
+                                                                 Coveralls::SimpleCov::Formatter
+                                                               ])
```
Comment (ndushay) [inline spec/spec_helper.rb]: this looks like bad whitespace added?

Comment (atz) [reply spec/spec_helper.rb]: @ndushay It's to align the opening and closing square brackets.  It is the expected behavior of that particular cop.

File: lyber-core.gemspec (hunk 1 above)
Comment (atz) [reply lyber-core.gemspec]: Literally haven't even heard of a couple of these people.

**Pull request merged**